packages/k8s/src/k8s/utils.ts

```diff
@@ -100,7 +100,9 @@ export function writeEntryPointScript(
 ): { containerPath: string; runnerPath: string } {
   let exportPath = ''
   if (prependPath?.length) {
-    exportPath = `export PATH=${prependPath.join(':')}:$PATH`
+    const prepend =
```

Comment:
Is using $PATH fine for k8s? I think it's still going to be the $PATH of the runner container (runner pod container?)

Reply:
I think it is, because we are writing an entrypoint script there, and not invoking container from a host machine.
That way, entrypoint would export envs and invoke a process which would have environment variables set. In Docker, we are invoking a command from our host which is causing the problem, but here, we have an entrypoint which is executed inside the container on startup, loading envs and calling the process.